src/coreclr/jit/llvm.cpp

```diff
@@ -2475,7 +2466,7 @@ void Llvm::lowerToShadowStack()
         _currentRange = &LIR::AsRange(_currentBlock);
         for (GenTree* node : CurrentRange())
         {
-            if (node->OperIs(GT_STORE_LCL_VAR, GT_LCL_VAR, GT_LCL_VAR_ADDR, GT_LCL_FLD_ADDR))
+            if (node->OperIs(GT_STORE_LCL_VAR, GT_LCL_VAR, GT_LCL_VAR_ADDR, GT_LCL_FLD_ADDR, GT_LCL_FLD))
```

Comment:
```suggestion
            if (node->OperIs(GT_STORE_LCL_VAR, GT_LCL_VAR, GT_LCL_FLD, GT_LCL_VAR_ADDR, GT_LCL_FLD_ADDR))
```
(Symmetry...)

I guess the only thing left to implement here is `STORE_LCL_FLD` and then we can convert this to `OperIsLocal() || OperIsLocalAddr()`.